Kiosk/ServiceHelpers/InkRecognizer.cs

```diff
@@ -0,0 +1,139 @@
+﻿// 
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// 
+// Microsoft Cognitive Services: http://www.microsoft.com/cognitive
+// 
+// Microsoft Cognitive Services Github:
+// https://github.com/Microsoft/Cognitive
+// 
+// Copyright (c) Microsoft Corporation
+// All rights reserved.
+// 
+// MIT License:
+// Permission is hereby granted, free of charge, to any person obtaining
+// a copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to
+// permit persons to whom the Software is furnished to do so, subject to
+// the following conditions:
+// 
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+// 
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// 
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Numerics;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.Data.Json;
+using Windows.Graphics.Display;
+using Windows.UI.Input.Inking;
+
+namespace ServiceHelpers
+{
+    public class InkRecognizer
+    {
+        string inkRecognitionUrl;
+        HttpClient httpClient;
+
+        private IDictionary<uint, InkStroke> StrokeMap { get; set; }
+        private string LanguageCode { get; set; }
+
+        public InkRecognizer(string subscriptionKey, string endpoint, string inkRecognitionUrl)
+        {
+            httpClient = new HttpClient() { BaseAddress = new Uri(endpoint) };
+            httpClient.DefaultRequestHeaders.Add("Ocp-Apim-Subscription-Key", subscriptionKey);
+
+            this.inkRecognitionUrl = inkRecognitionUrl;
+
+            StrokeMap = new Dictionary<uint, InkStroke>();
```

Comment:
>StrokeMap [](start = 12, length = 9)

I looked at a couple of existing files and they are using the this. style for member variables.  you should match whatever is the predominant style